### ISSUE TITLE
No compatible binary build found for this package when updating to Meteor 1.3.4.1

### ISSUE THREAD

**Issue by indesignlatam** [lookback/meteor-emails#68]
Hi, first of all thanks for the great job on this package.

I just updated one of my projects to the latest Meteor version 1.3.4.1 and im getting the next error, No compatible binary build found for this package.

It happens only when installing 0.7.4 on version 0.7.3 everything is OK

**Comment by MartinBucko** [comment]
+1 on ubuntu 

```
This project is already at Meteor 1.3.4.1, the latest release.
=> Errors while upgrading packages:           

While checking for lookback:emails@0.7.4:
error: No compatible binary build found for this package. Contact the package author and ask them to publish it for
your platform.
```

**Comment by johanbrook** [comment]
Thanks, @MartinBucko and @indesignlatam. I don't know why the binary dependencies somehow needed to be rebuilt for 0.7.4. But I built them again for all architectures now (OS X, Windows, Linux 32bit, Linux 64bit).

It works out of the box on my OS X machine, but I can't sadly test on Linux – would you be able to try again? Thanks!

**Comment by steve-ross** [comment]
I just got this error trying to upgrade to 1.4.0.1 from 1.3.2.4

**Comment by pierrelouisd4j** [comment]
@johanbrook I confirm it's fixed on Linux too ! (at least up to Meteor 1.3.5.1)
